docs/guides/custom-redirects.mdx

```diff
@@ -71,7 +71,12 @@ export default clerkMiddleware((auth, req) => {
 });
 
 export const config = {
-  matcher: ['/((?!.*\\..*|_next).*)', '/', '/(api|trpc)(.*)'],
+  matcher: [
+    // Skip Next.js internals and all static files, unless found in search params
+    '/((?!_next|[^?]*\\.(?:html?|css|js(?!on)|jpe?g|webp|png|gif|svg|ttf|woff2?|ico|csv|docx?|xlsx?|zip|webmanifest)).*)',
+    // Always run for API routes
+    '/(api|trpc)(.*)',
+  ],
```

Comment:
Do we want to replace with the commented version everywhere? At the moment, we have a commented version explaining the regex in the main docs, but in other places that it's referenced in code examples, we leave out the comments

Reply:
The quickstart guide didn't have comments before and I guess this is where most people learn about the matcher, that's why I included the comments there. Why woudln't we want the comments everywhere? People would probably just copy and paste the matcher when they find it I guess

Reply:
I say let's be consistent and keep the comments everywhere - we never know where exactly in the docs someone might be copying and pasting code from and comments don't hurt